package.json

```diff
@@ -15,6 +15,7 @@
     "@": "serverBuild/"
   },
   "dependencies": {
+    "bootstrap": "^4.2.1",
```

Comment:
I'm secretly a [bulma](https://bulma.io/documentation/) fan, but apparently bootstrap is cool again since v4.

Reply:
I'm openly a [foundation](https://foundation.zurb.com/sites/docs/) fan but totally undogmatic.
Would be happy with either framework that does good for us (the bootstrap answer to `sr-only` looked nasty but elaborate). Either way I'd prefer if we could preserve SOLAC, hence the mix-in proposal.